src/QMCWaveFunctions/ConstantOrbital.h

```diff
@@ -79,11 +79,11 @@ class ConstantOrbital: public OrbitalBase
     return FakeGradRatio;
   }
 
-  virtual RealType registerData(ParticleSet& P, BufferType& buf) {return 0.0;}
+  virtual void registerData(ParticleSet& P, WFBufferType& buf) {}
```

Comment:
Do we have any tests to indicate that ConstantOrbital is functional pre/post modification?

Reply:
This is introduced only for unit tests. My modification must be correct as the unit tests pass.